sl_opt.cpp:454:11: error: format not a string literal and no format arguments

Compilation fail during Debian package build on:

```
sl_opt.cpp
../../../../src/parsec/sl_opt.cpp: In function ‘void SYSs_CheckCommandLine(int, char**)’:
../../../../src/parsec/sl_opt.cpp:454:11: error: format not a string literal and no format arguments [-Werror=format-security]
  454 |   printf( options_invalid );
      |           ^~~~~~~~~~~~~~~
cc1plus: some warnings being treated as errors
make[3]: *** [Makefile:776: obj/x64/Release/sl_opt.o] Chyba 1
```

--- a/src/parsec/sl_opt.cpp
+++ b/src/parsec/sl_opt.cpp
@@ -451,7 +451,7 @@ void SYSs_CheckCommandLine( int argc, char **argv )
 
 	// exec all registered command line options
 	if ( !OPT_ExecRegisteredOptions( argc, argv ) ) {
-		printf( options_invalid );
+		printf( "%s", options_invalid );
 		exit( EXIT_SUCCESS );
 	}
 }

